src/coin_test/data/datasets.py

```diff
@@ -139,13 +141,17 @@ def split(
         timestamp: pd.Timestamp | None = None,
         length: pd.Timedelta | None = None,
         percent: float | None = None,
+        pre_name: str = "_pre",
+        post_name: str = "_post",
     ) -> tuple["Dataset", "Dataset"]:
         """Split Dataset into Pre and Post split Datasets.
 
         Args:
             timestamp: [Optional] Timestamp to split Dataset on
             length: [Optional] pd.Timedelta to specify length of the pre-split dataset
             percent: [Optional] float percentage of the data to split on
+            pre_name: Suffix to append to dataset name for pre section of the split
+            post_name: Suffix to append to dataset name for post section of the split
```

Comment:
Could this default to `_train` and `test`? Presumably this is the primary use case.

Reply:
So I thought about doing this... I would rather it be explicit if it is getting called by the user (I think this is). **Since the split doesn't actually enforce if you are using this as train or test data it feels awkward to force label it here**